src/tests/JIT/Regression/JitBlue/GitHub_69659/GitHub_69659_1.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+// In this issue, although we were not removing an unreachable block, we were removing all the code
+// inside it and as such should update the liveness information. Since we were not updating the liveness
+// information for such scenarios, we were hitting an assert during register allocation.
+public class Program
+{
+    public static ulong[] s_14;
+    public static uint s_34;
+    public static int Main()
+    {
+        var vr2 = new ulong[][]{new ulong[]{0}};
+        M27(s_34, vr2);
+        return 100;
+    }
+
+    public static void M27(uint arg4, ulong[][] arg5)
+    {
+        arg5[0][0] = arg5[0][0];
+        for (int var7 = 0; var7 < 1; var7++)
+        {
+            return;
+        }
+
+        try
+        {
+            s_14 = arg5[0];
+        }
+        finally
+        {
+            arg4 = arg4;
+        }
+    }
+}
```

Comment:
Sorry for the late review, but GitHub_XYZ refers to the dotnet/coreclr repo, so this should have been named Runtime_69659. Also it would be nice to name the class something like `Runtime_69659`, it makes it easier to track the test down on failures.

It seems there are a few tests incorrectly filed under GitHub_XYZ tag, so maybe something to consider cleaning up as part of a future change.

Reply:
> so this should have been named Runtime_69659

Ah, didn't realize that. I named them `_1` and `_2` because there were 2 separate issues that were filed in #69659. I think I forgot to have the right class name in one of those files.

Reply:
Ah sorry, `Runtime_69569_1` etc. sounds right then.